Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -733,6 +730,35 @@ private class PopupBlockingSettings: Setting {
     }
 }
 
+private class SaveLoginSettings: Setting {
```

Comment:
nit: there's a lot of duplicated code between all of the `UISwitch` based settings. Worth creating a subclass for SwitchSetting and then extending for `SaveLoginSettings`, `PopupBlockingSettings`, `ClosePrivateTabs`, `SendCrashReportsSetting` & `UseCompactTabLayoutSetting`?